Install-Kite.md

```diff
@@ -10,4 +10,22 @@ curl http://central.maven.org/maven2/org/kitesdk/kite-tools/{{site.version}}/kit
 chmod +x {{site.dataset-command}}
 ```
 
-The first command downloads the Kite tools JAR to a local file named _{{site.dataset-command}}_. The second changes the access rights on the JAR so that the classes in the JAR are executable.
+The first command downloads the Kite tools JAR to a local file named `{{site.dataset-command}}`. The second changes the access rights on the JAR so that the classes in the JAR are executable.
+
+Some versions of CDH come with `{{site.dataset-command}}` installed. However, it might not be the latest version. To check the version, from a terminal window, enter the following command.
+
+```
+{{site.dataset-command}} --version
+```
+
+If the version number doesn't match the version you downloaded, you have two options.
```

Comment:
This doesn't make sense to me in this context. These instructions are for verifying that the version installed is the version in use.